src/smokeTest/java/uk/gov/hmcts/reform/blobrouter/GetSasTokenTest.java

```diff
@@ -17,113 +18,138 @@
 
 import static io.restassured.config.SSLConfig.sslConfig;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
 
 public class GetSasTokenTest {
 
     private static final String SUBSCRIPTION_KEY_HEADER_NAME = "Ocp-Apim-Subscription-Key";
     private static final String PASSWORD_FOR_UNRECOGNISED_CLIENT_CERT = "testcert";
-
-    private final Config config = ConfigFactory.load();
+    private static final String SAS_TOKEN_ENDPOINT_PATH = "/token/bulkscan";
+    private static final String KEY_STORE_TYPE = "PKCS12";
+
+    private static Config config;
+    private static String apiGatewayUrl;
+    private static KeyStoreWithPassword validClientKeyStore;
+    private static KeyStoreWithPassword unrecognizedClientKeyStore;
+    private static KeyStoreWithPassword expiredClientKeyStore;
+    private static KeyStoreWithPassword notYetValidClientKeyStore;
```

Comment:
These key stores are only used in their respective tests. Best to reduce their scope to those tests.